tests/test_gcs.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from datasetinsights.io.gcs import GCSClient, gcs_bucket_and_path
+from datasetinsights.io.gcs import GCSClient, parse_gcs_location
 
 
 def test_gcs_client_warpper():
```

Comment:
Typo

Reply:
Also this test is doing multiple test cases. It should be refactored into multiple tests, upload/download single fie/download folder.